internal/plugins/helm/v1/init.go

```diff
@@ -143,14 +154,17 @@ func (p *initPlugin) Run() error {
 
 // Validate perform the required validations for this plugin
 func (p *initPlugin) Validate() error {
-	// Check if the project name is a valid namespace according to k8s
-	dir, err := os.Getwd()
-	if err != nil {
-		return fmt.Errorf("error to get the current path: %v", err)
+
+	// Check if the project name is a valid k8s namespace (DNS 1123 label).
+	if p.config.ProjectName == "" {
+		dir, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("error getting current directory: %v", err)
+		}
+		p.config.ProjectName = strings.ToLower(path.Base(dir))
```

Comment:
We should use `filepath` to handle Windows paths.
```suggestion
		p.config.ProjectName = strings.ToLower(filepath.Base(dir))
```